apps/server/src/modules/health/repo/healthcheck.repo.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common';
+import { EntityManager } from '@mikro-orm/mongodb';
+
+import { Healthcheck } from '../domain';
+import { HealthcheckEntity } from './entity';
+import { HealthcheckRepoMapper } from './healthcheck.repo.mapper';
+
+@Injectable()
+export class HealthcheckRepo {
+	constructor(private readonly em: EntityManager) {}
+
+	async upsertById(id: string): Promise<Healthcheck> {
+		const entity = await this.em.upsert(HealthcheckEntity, { id, updatedAt: new Date() });
+
+		return HealthcheckRepoMapper.mapHealthcheckEntityToDO(entity);
+	}
+}
```

Comment:
```suggestion
import { Injectable } from '@nestjs/common';
import { EntityManager } from '@mikro-orm/mongodb';

import { HealthCheck } from '../domain';
import { HealthCheckEntity } from './entity';
import { HealthCheckRepoMapper } from './health-check.repo.mapper';

@Injectable()
export class HealthCheckRepo {
	constructor(private readonly em: EntityManager) {}

	async upsertById(id: string): Promise<HealthCheck> {
		const entity = await this.em.upsert(HealthCheckEntity, { id, updatedAt: new Date() });

		return HealthCheckRepoMapper.mapHealthCheckEntityToDO(entity);
	}
}
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/9f7cb45820e06668af950204e05f82983c112e54 .